snapshot/generate_test.go

```diff
@@ -118,6 +149,10 @@ func hashData(input []byte) common.Hash {
 
 // Tests that snapshot generation with existent flat state.
 func TestGenerateExistentState(t *testing.T) {
+	if false {
```

Comment:
hmm??? false??
@jeongkyun-oh What about adding a comment here something like "replace it with true if you want to see logs"